csfieldguide/static/interactives/sorting-algorithms-configurator/scss/sorting-algorithms-configurator.scss

```diff
@@ -1,6 +1,5 @@
 // The interactive sticks awkwardly close to the left of a teacher panel
 // for some unknown reason
 #sorting-algorithms-configurator {
```

Comment:
Try adding the `d-inline` class to this div instead. Would be nicer if it works.

Reply:
It didn't work _and_ it added a scrollbar underneath

Reply:
my baaaad, I meant `d-inline-block`